pkg/cmd/tests/tests_run.go

```diff
@@ -434,9 +436,19 @@ func (o *RunOptions) run(ctx context.Context, streams genericclioptions.IOStream
 
 	wg.Wait()
 
+	for i, e := range cmdEntries {
+		exitCode := e.cmd.ProcessState.ExitCode()
+		switch exitCode {
+		case 0, types.GINKGO_FOCUS_EXIT_CODE:
+			break
+		default:
+			klog.ErrorS(errs[i], "Process failed", "ID", e.id, "Command", e.cmd.String(), "ProcessState", e.cmd.ProcessState.String(), "Logs", e.out.String())
+		}
+	}
+
 	err = apierrors.NewAggregate(errs)
 	if err != nil {
-		klog.ErrorS(err, "Processes encountered errors")
+		return fmt.Errorf("processes encountered errors: %w", err)
```

Comment:
I've looked into `exec.Wait` and it will return an error on non-zero codes so we'll have to do it a bit differently I think. Wanna try something like https://github.com/scylladb/scylla-operator/commit/7920c201d1e059ed758bbd2b78f3b1474c65338e ?

Reply:
done